reference/6/About/about_Special_Characters.md

```diff
@@ -112,6 +133,20 @@ The output from this command is:
 
 Column1         Column2         Column3
 
+UNICODE CHARACTER (`u{x})
+The Unicode escape sequence allows you to specify any Unicode character
```

Comment:
I like that.  I'll wait a bit, for any other updates, and submit a new PR. Thanks.